src/guides/debugging-an-example-smart-contract/index.md

```diff
@@ -606,28 +697,28 @@ Just as before, we'll reset the contract on the blockchain.
 1. Press `Enter` multiple times to cycle through the steps. Eventually you will see that the conditional leads to the `Odd()` event:
 
    ```solidity
-   Store.sol | 0x377bbcae5327695b32a1784e0e13bedc8e078c9c:
+   Store.sol:
 
-   10:   function set(uint x) public {
-   11:     myVariable = x;
-   12:     if (x % 2 == 0) {
-           ^^^^^^^^^^^^^^^^
+   9:   function set(uint x) public {
+   10:     myVariable = x;
+   11:     if (x % 2 == 0) {
+           ^^^^^^^^^^^^^^^^^
 
-   debug(develop:0x7f799ad5...)>
+   debug(develop:0x31d64ba6...)> n
 
-   Store.sol | 0x377bbcae5327695b32a1784e0e13bedc8e078c9c:
+   Store.sol:
+
+   10:     myVariable = x;
+   11:     if (x % 2 == 0) {
+   12:       emit Odd();
+                  ^^^^^
 
-   11:     myVariable = x;
-   12:     if (x % 2 == 0) {
-   13:       Odd();
-             ^^^^^
+   debug(develop:0x31d64ba6...)>
 
-   debug(develop:0x7f799ad5...)>
    ```
 
    **The problem is revealed.** The conditional is leading to the wrong event.
 
-
 ## Conclusion
 
 With the ability to debug your contracts directly within Truffle, you have even more power at your hands to make your smart contracts rock-solid and ready to deploy. Make sure to read more about Truffle Develop console and the debugger in the docs. If you have any questions, please join our [community Gitter channel](https://gitter.im/ConsenSys/truffle) where hundreds of fellow Trufflers congregate to answer your questions!
```

Comment:
I'm thinking we could remove the part about heading over to our Gitter and recommend people open an issue on Github if they have any questions. Thoughts?

Reply:
Yeah, I like this point, agreed!